python/cuspatial/cuspatial/_lib/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@
 # =============================================================================
 
 set(cython_sources
+    allpairs_multipoint_equals_count.pyx
```

Comment:
cython files are names mismatch!

Reply:
And so is all python bindings.

Reply:
Seems like the wrong commit, those were all updated iirc.

Reply:
Ok weird, not sure how I missed this. I remember renaming everything...

Reply:
Oh I remember now I finished the python refactor in the follow-up PR so I could start using it.

Reply:
Fixed